site/src/theme/darkBlue/branding.ts

```diff
@@ -0,0 +1,19 @@
+import type { Branding } from "../branding";
+import colors from "../tailwindColors";
+
+export default {
+	paywall: {
+		premium: {
+			background: colors.violet[950],
+			divider: colors.violet[900],
+			border: colors.violet[400],
+		},
+	},
+	badge: {
+		premium: {
+			background: colors.violet[950],
+			border: colors.violet[400],
+			text: colors.violet[50],
+		},
+	},
```

Comment:
you're being specific again :p

the two values that `paywall` and `badge` share–`background` and `border`– are identical. why do they need to be duplicated? I think this should just be flatten. it's not the job of the theme to cater to _specific_ UI components, it's job is to provide general styles.

Reply:
I still accustomed to other projects where the theme can be specific to UI components, here is an example, https://github.com/supabase/supabase/blob/4dfadb6c03bf832ba875270ccc81f91d278f0df4/packages/ui/build/css/themes/dark.css

I agree here it makes sense to flatten it especially since I removed the enterprise colors.